MakingItBetterG55/src/application/Main.java

```diff
@@ -3,20 +3,90 @@
 import javafx.application.Application;
 import javafx.stage.Stage;
 import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.RowConstraints;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.KeyEvent;
-import javafx.scene.layout.GridPane;
 import javafx.scene.layout.Pane;
 
-
 public class Main extends Application {
 	@Override
 	public void start(Stage primaryStage) {
 		try {
+			GridPane homeScreen = new GridPane();
+			
+			//Set this to your own URL or local directory for the background image file :)
+			homeScreen.setStyle("-fx-background-image: url('file:///C:/Users/Benjamin%20Keyes/Documents/JavaFX%20Workspace/MakingItBetter/src/images/woodbackground.jpg');");
+			
+			Scene homeScene = new Scene(homeScreen, 1200, 800);
+
+			homeScreen.setGridLinesVisible(false);
+	        final int numCols = 40;
+	        final int numRows = 40;
+	        for (int i = 0; i < numCols; i++) {
+	            ColumnConstraints colConst = new ColumnConstraints();
+	            colConst.setPercentWidth(100.0 / numCols);
+	            homeScreen.getColumnConstraints().add(colConst);
+	        }
+	        for (int i = 0; i < numRows; i++) {
+	            RowConstraints rowConst = new RowConstraints();
+	            rowConst.setPercentHeight(100.0 / numRows);
+	            homeScreen.getRowConstraints().add(rowConst);         
+	        }
+	        Text title = new Text("Making It Better");
+	        title.setFont(Font.font("SansSerif", FontWeight.BOLD, 100));
+	        title.setStyle("-fx-fill: black; -fx-stroke: white; -fx-stroke-width: 4px;");
+
+			Button playButton = new Button("Play");
+			playButton.setStyle("-fx-cursor: hand; -fx-border-color: #152546; -fx-border-width: 14px; -fx-background-color: #536F7B; -fx-text-fill: white;");
+			playButton.setFont(Font.font("SansSerif", FontWeight.BOLD, 40));
+			playButton.setPrefSize(320, 29);
+			
+			playButton.setOnAction(event -> {
+			//Connecting to a blank scene for now - Replace with Start Screen
+			GridPane tempScene = new GridPane();
+			Scene newScene = new Scene(tempScene, 1200, 800);
+		    Stage stage = (Stage)playButton.getScene().getWindow();
+		    stage.setScene(newScene);
+			});
+			
+			Button quitButton = new Button("Quit");
+			quitButton.setFont(Font.font("SansSerif", FontWeight.BOLD, 40));
+			quitButton.setStyle("-fx-cursor: hand; -fx-border-color: #152546; -fx-border-width: 14px; -fx-background-color: #536F7B; -fx-text-fill: white;");
+			quitButton.setPrefSize(320, 29);
+			//Adding the event to close the menu
+			quitButton.setOnAction(event -> {
+			    Scene tempPlaceholder = quitButton.getScene();
+			    homeScene.getWindow().hide();
```

Comment:
should we not use something like System.exit(0) instead of hide here?

Reply:
Done.